### PR TITLE
Added PR Template

### PR DESCRIPTION
**Pull request by LukasLJL**
# Skiosa - PR
- [x] documentation can't be split up into smaller logical units

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Skiosa - PR
+- [ ] documentation can't be split up into smaller logical units
+- [ ] document is formated
```
Comment (jeppard) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Latexindent not working

**Review by jeppard — CHANGES_REQUESTED** [review]
Add Latexindent that latex document can be formated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (LukasLJL) [reply .github/PULL_REQUEST_TEMPLATE.md]: removed from template

**Review by jeppard — COMMENTED** [review]
lgtm

**Review by jeppard — APPROVED** [review]

**Pull request merged**